metrics/cli/lib/interfaces/cli/cli.dart

```diff
@@ -1,8 +1,32 @@
 // Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:io';
+
+import 'package:process_run/process_run.dart' as cmd;
+
 /// A base class for CLIs that provides common methods for them.
 abstract class Cli {
+  /// An executable name of this CLI.
```

Comment:
```suggestion
  /// A name of the executable for this CLI.
```
Would it be a bit more clear?